extensions/amp-date-picker/0.1/test/validator-amp-date-picker.html

```diff
@@ -142,5 +148,11 @@
   <!-- Invalid: single amp-date-picker with minimum nights -->
   <amp-date-picker layout="fixed-height" height="360" minimum-nights="0">
   </amp-date-picker>
+  <!-- Invalid: single amp-date-picker with start-date attribute -->
```

Comment:
I believe this should be `Invalid: range ...` and not `Invalid: single`.

Reply:
It looks correct to me. It's `invalid: single` because the amp-date-picker below doesn't have `type="range"` attribute and `start-date` is only valid on a date picker with `type="range"`

Reply:
Ah, okay. Just checked but it's also giving the invalid layout error. Is there a layout that can be used that would report that `type="range"` is missing? Otherwise this test isn't really testing what you think it is.

Reply:
hmm I'm not sure why the validator is applying the `mode=overlay` rules here. amp-date-picker is meant to be `mode=static` by default. It's even doing it here where `mode=static` is explicitly specified. https://github.com/ampproject/amphtml/pull/18350/files#diff-69f02f4e4ab10df1bc26b10472b8a076R118

Reply:
This stems from @Gregable's comment here https://github.com/ampproject/amphtml/pull/14109#issuecomment-376256275 so I don't think it's a blocker